### PR TITLE
Added log handler

### PR DESCRIPTION
**Pull request by Kelvin4664**

### CODE REVIEW

File: honeybadger/contrib/logger.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        return {
+            k: v for (k, v) in record.__dict__.items()
+            if k not in DEFAULT_IGNORED_KEYS
+        }
+  
+    def emit(self, record):
+        
+        try:
+            self.honeybadger.notify(
+                error_class = record.levelname,
```
Comment (joshuap) [inline honeybadger/contrib/logger.py:44]: Let's make this "[record.levelname] Log"

**Review by joshuap — COMMENTED** [review]
Looking good. Can you tell me what the backtrace section of the reported error looks like for logs? Is there any way to get a traceback for where the `logging` call was made?

Also, we need README documentation for how to use this, as well as a changelog entry.

**Review by joshuap — COMMENTED** [review]
@Kelvin4664 A few  questions:

1. So if I'm reading the docs correctly, this handler can be added to any existing logger, in which case the existing logger's log level will be used to report exceptions? So if the logger is configured with WARNING, then INFO logs will not be reported to Honeybadger on that logger?
2. Is there a way to configure django like the [sample config](https://github.com/honeybadger-io/honeybadger-python/issues/82#issue-879798452) I provided in #82? Can we document that as well?
3. (just copying this from my prior comment to resurface it) Can you tell me what the backtrace section of the reported error looks like for logs? Is there any way to get a traceback for where the logging call was made?

**Comment by Kelvin4664** [comment]
> @Kelvin4664 A few questions:
> 
> 1. So if I'm reading the docs correctly, this handler can be added to any existing logger, in which case the existing logger's log level will be used to report exceptions? So if the logger is configured with WARNING, then INFO logs will not be reported to Honeybadger on that logger?
> 2. Is there a way to configure django like the [sample config](https://github.com/honeybadger-io/honeybadger-python/issues/82#issue-879798452) I provided in [Add a log handler #82](https://github.com/honeybadger-io/honeybadger-python/issues/82)? Can we document that as well?
> 3. (just copying this from my prior comment to resurface it) Can you tell me what the backtrace section of the reported error looks like for logs? Is there any way to get a traceback for where the logging call was made?

1. Yes
2. Yes.. Readme has been updated
3. Yes. We have a traceback just like we would for an exception. Screenshot below
image

File: honeybadger/contrib/logger.py [hunk 2]
```diff
@@ -28,6 +28,7 @@ def __init__(self, api_key):
 
         self.honeybadger = Honeybadger()
         self.honeybadger.configure(api_key=api_key)
+        print(api_key)
```
Comment (joshuap) [inline honeybadger/contrib/logger.py:31]: Is this leftover from debugging?

```suggestion
```

**Review by joshuap — APPROVED** [review]
Looks good!

File: honeybadger/contrib/logger.py (hunk 2 above)
Comment (joshuap) [reply honeybadger/contrib/logger.py:31]: I'll just commit this change.

File: README.md [hunk 3]
```diff
@@ -273,6 +273,46 @@ import logging
...
+or using Dict Config:
+```python
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'honeybadger': {
+            'level': 'ERROR',
+            'class': 'honeybadger.contrib.logger.HoneybadgerHandler',
+            'api_key': '**YOUR API KEY**',
```
Comment (joshuap) [inline README.md:302]: Can we use the existing api_key if Honeybadger was already configured globally with `honeybadger.configure`?

**Review by joshuap — COMMENTED** [review]
@Kelvin4664 I forgot, one more question.

File: README.md (hunk 3 above)
Comment (Kelvin4664) [reply README.md:302]: No.. not without some heavy hack that i don't think will be worth it. Or having separate log handlers for the different frameworks. this is the safest bet and i got the hint from rollbar's approach.

**Pull request merged**